src/user/system/CaptainManager.ts

```diff
@@ -648,8 +663,44 @@ class CaptainManager {
                             key: 'PUSHBULLET_DEFAULT_EMAIL',
                             value: netDataInfo.data.pushBullet.fallbackEmail,
                         })
+                        envVars.push({
+                            key: 'SEND_PUSHBULLET',
+                            value: 'YES',
+                        })
                     }
-
+                    
+                    // For Discord Block
+                    if (netDataInfo.data.discord) {
+                        envVars.push({
+                            key: 'DISCORD_WEBHOOK_URL',
```

Comment:
The new env vars need to be reflected in the frontend as well. They won't be used if they're not surfaced on the frontend.

Reply:
They were present in the run.sh I was thinking they must be also in the frontend.